site/src/components/gradients.md

```diff
@@ -30,6 +30,22 @@ Because gradient transitions take up the entire width of a particular element, i
 <h3 class="font-size-h3 font-weight-bold">
 	A vivid gradient from <span class="gradient-text-vivid">purple to blue</span>
 </h3>
+<h3 class="font-size-h3 font-weight-bold">
+	A bright gradient from <span class="gradient-text-red-purple">red to purple</span>
+</h3>
+```
+
+## Direction of gradient
+
+You can specify for the gradient to transition vertically instead of horizontally by appending `-bottom` to the end of the gradient class. By default, the gradient moves from left to right, so you do not need to specify `-right`.
```

Comment:
By default, the gradient moves from left to right. To transition gradient vertically instead, append `-bottom` to the end of the gradient atomic - `gradient-text-<colorname>-bottom`.